.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,34 @@
+name: Pull Request
+on: [pull_request, push]
```

Comment:
You can also specify that this should only run on PRs or pushes to individual branches if you want: https://github.com/lyft/clutch/blob/main/.github/workflows/frontend.yml#L3-L8